packages/client/src/views/OfficeHome/inProgress/InProgress.tsx

```diff
@@ -149,9 +149,6 @@ export class InProgressComponent extends React.Component<
   }
 
   componentWillUnmount() {
-    window.removeEventListener('popstate', function (event) {
-      window.history.pushState(null, document.title, window.location.href)
-    })
```

Comment:
@favour234 Apologies for not being able to explain to you clearly. It is definitely the right thing to do to remove eventListeners on unmount. What I was trying to say is to extract the function into a class method like:
```ts
  popListener() {
    window.history.pushState(null, document.title, window.location.href)
  }
``` 
Then use this method in the eventListeners like this:
```ts
  window.addEventListener('popstate', this.popListener)
```
```ts
  window.removeEventListener('popstate', this.popListener)
```

Reply:
alright..its clearer now

Reply:
updated.